website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Finding Cross Account AWS EventBridge Usage
+tag: security
+date: 2022/11/25
+description: >-
+  How a CloudQuery customer wrote custom queries to find cross account AWS EventBridge Usage to help their teams migrate to new AWS EventBridge security features to increase security and compliance.
```

Comment:
This is quite a long sentence with a lot of "to"s, can we maybe simplify / split it into multiple sentences? 😄

Reply:
Split, thanks!